### PR TITLE
Profile arabic style bugs

### PR DESCRIPTION
**Pull request by Jamil-Bailony**
- [x] fix the arrow direction when Arabic language chosen

- [x] add the corresponding styles

- Arabic language:
image

- English language:
image

### CODE REVIEW

File: src/containers/ProfilePage/ToggleButton.jsx [hunk 1]
```diff
@@ -1,23 +1,52 @@
 import React from "react";
 import { useAccordionToggle, AccordionContext } from "react-bootstrap";
+import i18n from "i18next";
 
 function getArrowClass(isCurrentElementSelected) {
-  return isCurrentElementSelected ? "arrow down" : "arrow right";
+  let arrowClass;
+
+  if (isCurrentElementSelected) {
```
Comment (louisrli) [inline src/containers/ProfilePage/ToggleButton.jsx]: pls just make it one flat level of if-conditions, the nested if is impossible to tell whats going on

Comment (Jamil-Bailony) [reply src/containers/ProfilePage/ToggleButton.jsx]: how about this:
```
  if (isCurrentElementSelected) {
    // if Arabic, make the margin from left
    // or make it default, margin from right
    arrowClass = i18n.language === "ar"
      ? "arrow down-rtl"
      : "arrow down";
  } else if (i18n.language === "ar") {
    arrowClass = "arrow left";
  } else {
    arrowClass = "arrow right";
  }

  return arrowClass;
}
```

is using ternary operator okay? I am afraid that if would be too long and that affect the readability. @louisrli

Comment (Jamil-Bailony) [reply src/containers/ProfilePage/ToggleButton.jsx]: >  louisrli: yeah, it's fine, change it to approve, you are doing good work, I like it

Thank you @louisrli, I know there is no time to think ;)

**Review by jennetturkkan — APPROVED** [review]

File: src/containers/ProfilePage/ToggleButton.jsx (hunk 1 above)
Comment (louisrli) [reply src/containers/ProfilePage/ToggleButton.jsx]: if you're going to do that it makes more sense to split it by language. naturally we think in terms fo what language the page is first and then what happens

if (language === "ar") {
 arrow = isSelected ? foo : bar
}
else {
 arrow = isSelected ?
}

**Review by mohammad-jundi — APPROVED** [review]

File: src/containers/ProfilePage/ToggleButton.jsx (hunk 1 above)
Comment (Jamil-Bailony) [reply src/containers/ProfilePage/ToggleButton.jsx]: done!

**Comment by Jamil-Bailony** [comment]
if you can take a look again please @louisrli

File: src/containers/ProfilePage/ToggleButton.jsx [hunk 2]
```diff
@@ -1,23 +1,45 @@
...
 import { useAccordionToggle, AccordionContext } from "react-bootstrap";
+import i18n from "i18next";
 
 function getArrowClass(isCurrentElementSelected) {
-  return isCurrentElementSelected ? "arrow down" : "arrow right";
+  let arrow;
+
+  if (i18n.language === "ar") {
+    // if Arabic, make the margin from left
+    arrow = isCurrentElementSelected ? "arrow down-rtl" : "arrow left";
```
Comment (louisrli) [inline src/containers/ProfilePage/ToggleButton.jsx]: return isCurrent ? x : y

same with line 12

Comment (Jamil-Bailony) [reply src/containers/ProfilePage/ToggleButton.jsx]: done!

Comment (louisrli) [reply src/containers/ProfilePage/ToggleButton.jsx]: its not

Comment (Jamil-Bailony) [reply src/containers/ProfilePage/ToggleButton.jsx]: forgot to push, sorry @louisrli

**Review by louisrli — APPROVED** [review]

**Pull request merged**